server/spec/test_mneme_api_response.rb

```diff
@@ -27,6 +32,8 @@ def setup
     # allow for file by file override
     #logger.level=Logger::ERROR
     #logger.level=Logger::DEBUG
+    @mock_today_epoch = 1417409940
```

Comment:
It's there to give a consistent date for testing.  Testing will be easier if the value doesn't depend on when the test is run.